_source/logzio_collections/_security-sources/auth0.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Ship Auth0 events
+logo:
+  logofile: auth0.png
+  orientation: vertical
+data-source: Auth0
+templates: ["no-template"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - identity
+order: 810
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Custom log stream](#webhooks)
+* [Logstash extension](#logstash)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="webhooks">
+
+### Ship events data from your Auth0 account to Logz.io
+
+Deploy this integration to ship Auth0 events from your Auth0 account to Logz.io using custom log stream via webhooks.
+
+**Before you begin, you'll need**: an active account with Auth0
+
+<div class="tasklist">
+
+#### Setup a custom log stream using webhooks
+
+##### Create a custom log stream
+
+1. Log in to your Auth0 account, navigate to **Auth0 Dashboard > Monitoring > Streams**, and select **Create Stream**.
+  
+2. On the **New Event Stream**, select **Custom Webhook**.
+
+   The following screen appears:
+   ![Create stream](https://dytvr9ot2sszz.cloudfront.net/logz-docs/auth0/Auth0-logzio-integration.png)
+  
+3. In the **Name** field, enter **Logz.io integration** and select **Create**.
+
+
+##### Configure the required parameters
+  
+The **Settings** tab of the created event stream is displayed after you create the stream.
+
+Configure the required parameters as follows:
+
+![Configure stream](https://dytvr9ot2sszz.cloudfront.net/logz-docs/auth0/webhook-auth0.png)
+
```

Comment:
Next time there are this many fields to configure, maybe it would be worth it to present them as a table...